georiviere/watershed/mixins.py

```diff
@@ -13,3 +13,7 @@ def zoning_property(self):
     @property
     def watersheds(self):
         return uniquify(intersecting(Watershed, self.zoning_property, distance=0))
+
+    @property
+    def watersheds_ordered_watershed_type(self):
+        return sorted(self.watersheds, key=lambda x: x.watershed_type.name)
```

Comment:
you can't order by type__name ?

Reply:
No because it's already a list because we order by the distance to the source